arex-instrumentation/dynamic/arex-dynamic-common/src/test/java/io/arex/inst/dynamic/common/DynamicClassExtractorTest.java

```diff
@@ -572,6 +572,20 @@ void testProceedingJointPoint() throws Throwable {
         assertEquals("[\"java.lang.String\"]", requestType.get(extractor));
     }
 
+    @Test
+    void testProceedingJointPointParamContainNUll() throws Throwable {
```

Comment:
Please change the unit test about `ArrayUtils.toString` method, there are no changes here, keep the unit test.